BEIMA.Client/src/pages/DeviceTypes/DeviceTypePage.js

```diff
@@ -0,0 +1,216 @@
+import { useState, useEffect } from "react"
+import {ItemCard} from "../../shared/ItemCard/ItemCard"
+import styles from './DeviceTypePage.module.css'
+import { useOutletContext } from 'react-router-dom';
+import { Form, Card, Button, FormControl} from "react-bootstrap";
+import { IoAdd } from "react-icons/io5";
+import { v4 as uuidv4 } from 'uuid';
+
+const DeviceTypePage = () => {
+  const [deviceType, setDeviceType] = useState(null)
+  const [loading, setLoading] = useState(true)
+  const [setPageName] = useOutletContext();
+
+  useEffect(() => {
+    setPageName('View Device Type')
+  },[setPageName])
+
+
+  const mockCall = async () => {
+    const sleep = ms => new Promise(resolve => setTimeout(resolve, ms))
+    await sleep(1000)
+    var data = {
```

Comment:
We should use 'let' instead of 'var' inside of functions, var sets the variable globally whereas let sets the variable within the given block.

Reply:
They should all be let now